api/proto/v1/teleworker.proto

```diff
@@ -0,0 +1,93 @@
+syntax = "proto3";
+
+option go_package = "internal/api/v1";
+
+package v1;
+
+service TeleWorker {
+  rpc Start(StartRequest) returns (StartResponse);
+  rpc Stop(StopRequest) returns (StopResponse);
+  rpc Status(StatusRequest) returns (StatusResponse);
+  rpc Stream(StreamRequest) returns (stream StreamResponse);
+}
+
+// UUID is used as a job identifier.
+message UUID {
```

Comment:
Nit: Is defining a separate message for the job ID really necessary? Can just use a string in messages requiring job ID for simplicity.

Reply:
Agreed, just wanted to explicitly show in here that uuids are used as id, but this can be simplified